app/src/ui/components/offers/ExpandedOffer.vue

```diff
@@ -56,13 +72,21 @@ const minMaxCryptoStr = computed(() => {
   return [`${symbol} ${min}`, `${symbol} ${max}`]
 })
 
-function newTrade() {
+async function newTrade() {
+  const telegramHandle = removeTelegramURLPrefix(telegram.value) as string
+  const profile_taker_encrypt_key = secrets.value.publicKey
```

Comment:
Can we please rename this to something like `profile_taker_encryption_key`, `profile_taker_pub_key` or even `taker_profile_pub_key`?